[2021-04-21 20:15:40,380] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: docker_sample.docker_op_tester 2021-04-19T10:10:00+00:00 [queued]>
[2021-04-21 20:15:40,411] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: docker_sample.docker_op_tester 2021-04-19T10:10:00+00:00 [queued]>
[2021-04-21 20:15:40,414] {taskinstance.py:1042} INFO - 
--------------------------------------------------------------------------------
[2021-04-21 20:15:40,416] {taskinstance.py:1043} INFO - Starting attempt 1 of 2
[2021-04-21 20:15:40,417] {taskinstance.py:1044} INFO - 
--------------------------------------------------------------------------------
[2021-04-21 20:15:40,439] {taskinstance.py:1063} INFO - Executing <Task(DockerOperator): docker_op_tester> on 2021-04-19T10:10:00+00:00
[2021-04-21 20:15:40,446] {standard_task_runner.py:52} INFO - Started process 662 to run task
[2021-04-21 20:15:40,459] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'docker_sample', 'docker_op_tester', '2021-04-19T10:10:00+00:00', '--job-id', '277', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/dockerOperator.py', '--cfg-path', '/tmp/tmpdigdcleu', '--error-file', '/tmp/tmp2l9qk9nj']
[2021-04-21 20:15:40,461] {standard_task_runner.py:77} INFO - Job 277: Subtask docker_op_tester
[2021-04-21 20:15:40,557] {logging_mixin.py:104} INFO - Running <TaskInstance: docker_sample.docker_op_tester 2021-04-19T10:10:00+00:00 [running]> on host 5af2b1cab26e
[2021-04-21 20:15:40,629] {taskinstance.py:1257} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_EMAIL=airflow@example.com
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=docker_sample
AIRFLOW_CTX_TASK_ID=docker_op_tester
AIRFLOW_CTX_EXECUTION_DATE=2021-04-19T10:10:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-04-19T10:10:00+00:00
[2021-04-21 20:15:40,645] {docker.py:224} INFO - Starting docker container from image centos:latest
[2021-04-21 20:16:11,949] {taskinstance.py:1166} INFO - Marking task as SUCCESS. dag_id=docker_sample, task_id=docker_op_tester, execution_date=20210419T101000, start_date=20210421T201540, end_date=20210421T201611
[2021-04-21 20:16:12,024] {taskinstance.py:1220} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2021-04-21 20:16:12,122] {local_task_job.py:146} INFO - Task exited with return code 0
